packages/core/src/transactions/TransactionRequest.ts

```diff
@@ -11,7 +12,8 @@ type TransactionRequestDataRpc = any
 type TransactionRequestDataWeb3 = any
 type TransactionRequestDataEthers = any
 
-export default class TransactionReques {
+export default class TransactionRequest {
+  readonly from!: string
```

Comment:
@bpierre I think we were missing the `from` on the specs. There was a particular reason why you didn't include it?

Reply:
Let’s add it yes :+1:  [Ethers doesn’t need it](https://docs.ethers.io/ethers.js/html/api-providers.html#transaction-request) and I was thinking that we could get this information later for the others… but it wouldn’t work because we don’t store it anywhere else.